src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -29,11 +29,21 @@ public WinRmToolResponse executeScript(List<String> commands) {
         return responseToWinRmToolResponse(response);
     }
 
+    public WinRmToolResponse executeCommand(String command) {
+        Response response = session.run_cmd(command);
+        return responseToWinRmToolResponse(response);
+    }
+
     public WinRmToolResponse executePs(List<String> commands) {
         Response response = session.run_ps(compileScript(commands));
         return responseToWinRmToolResponse(response);
     }
 
+    public WinRmToolResponse executePsCommand(String command) {
```

Comment:
What do you thing about renaming it only for `Ps`
and keep `executeCommand(String) and `executeCommand(List<String>)` for the "Command Prompt" commands? Really we do not `executeScript` no script was being composed here so I think it is wrong.